jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -68,14 +70,13 @@ class OutlierDetectionStep(Step):
         good_bits = string(default="~DO_NOT_USE")  # DQ flags to allow
         scale_detection = boolean(default=False)
         search_output_file = boolean(default=False)
-        allowed_memory = float(default=None)  # Fraction of memory to use for the combined image.
-        in_memory = boolean(default=False)
+        allowed_memory = float(default=None)  # Fraction of memory to use for the combined image
+        in_memory = boolean(default=True)
```

Comment:
For use in the operations environment we want `in_memory` to default to `False`, in order to keep processing runs from throwing out-of-memory errors. So this should be set back to the original value. With `save_intermediate_results` also defaulting to `False`, those files should not be left behind in the operations environment.